src/DominationTurn.js

```diff
@@ -36,7 +36,40 @@ class DominationTurn extends Turn {
     //surveys saved will allow for points to be added once dom ROUND is complete
   }
 
+  capitalizeGuess(guess) {
+    return guess.toUpperCase();
+  }
+
+  checkGuess(guess) {
+    return this.currentRound.answerStrings.includes(this.capitalizeGuess(guess));
+  }
+
+  spliceAnswers(guess) {
+    let i = this.currentRound.answerStrings.findIndex(answer => answer === this.capitalizeGuess(guess))
+    this.currentRound.answerStrings[i] = 'false';
+    this.currentRound.endRound();
+  }
```

Comment:
Are we using this splicing method again? or is it creeping from past commits?

Reply:
I just copied some of the methods over from the RegularTurn into DominationTurn, just to have something to mess with. Once I get it working I'll change them.